packages/utilities/psammead-rich-text-transforms/src/candy-xml.js

```diff
@@ -9,7 +9,7 @@ const attributeTags = ['bold', 'italic'];
 const supportedXmlNodeNames = ['paragraph', 'link', 'url', ...attributeTags];
 
 const isXmlNodeSupported = node => {
-  if (path(['constructor', 'name'], node) === 'XmlTextNode') {
+  if (path(['text'], node)) {
```

Comment:
Can we use the type from here? https://github.com/nfarina/xmldoc/blob/master/lib/xmldoc.js#L234 It shouldnt get lost when minified like the name